packages/migration-sdk-viem/src/positions/supply/aaveV3Optimizer.supply.ts

```diff
@@ -71,10 +71,10 @@ export class MigratableSupplyPosition_AaveV3Optimizer
     if (aaveV3OptimizerMigrationAdapter == null)
       throw new Error("missing aaveV3OptimizerMigrationAdapter address");
 
-    const migrationAddresses =
-      MIGRATION_ADDRESSES[chainId][MigratableProtocol.aaveV3Optimizer];
+    const aaveV3OptimizerAddresses =
+      migrationAddresses[chainId]?.[MigratableProtocol.aaveV3Optimizer];
 
-    if (!migrationAddresses) throw new UnsupportedChainIdError(chainId);
+    if (!aaveV3OptimizerAddresses) throw new UnsupportedChainIdError(chainId);
```

Comment:
shouldn't we expose a `getMigrationAddresses` that works the same as `getChainAddresses`?

Reply:
Indeed but less prio